docs/ref/settings.txt

```diff
@@ -2062,6 +2062,23 @@ non-zero value.
     :setting:`SECURE_HSTS_SECONDS`) break your site. Read the
     :ref:`http-strict-transport-security` documentation first.
 
+.. setting:: SECURE_HSTS_PRELOAD
+
+``SECURE_HSTS_PRELOAD``
+----------------------------------
```

Comment:
underline length should match heading length

Also, add a `.. versionadded:: 1.11` annotation here as well as a mention in `docs/releases/1.11.txt`.